docusaurus.config.js

```diff
@@ -134,15 +134,15 @@ export default {
           // Learn: Working with Assets Section
           {
             from: '/learn/smart-contract-developerss/retrieve-token-type',
-            to: '/learn/assets/retrieve-token-type',
+            to: '/learn/digital-assets/retrieve-token-type',
```

Comment:
I see the line above has 2 s:

```
smart-contract-developerss
```

weird - i think one is enough. I guess the current redirect is not good.

Reply:
good catch!